sass/util/_z-index.scss

```diff
@@ -0,0 +1,3 @@
+.z5 {
```

Comment:
do we need this? can we change to z4 (and the other to z3)? if not, let's at least make it one line